src/js/lib/addon.ts

```diff
@@ -58,9 +61,36 @@ export interface TextEmbeddingPipelineWrapper {
   embedDocumentsSync(documents: string[]): EmbeddingResults;
 }
 
+export interface VLMPipeline {
+  new (): VLMPipeline;
+  init(
+    modelPath: string,
+    device: string,
+    ovProperties: VLMPipelineProperties,
+    callback: (err: Error | null) => void,
+  ): void;
+  generate(
+    prompt: string,
+    images: Tensor[] | undefined,
+    videos: Tensor[] | undefined,
+    streamer: ((chunk: string) => StreamingStatus) | undefined,
+    generationConfig: GenerationConfig | undefined,
```

Comment:
Why these arguments are explicitly include undefined type? They can't be optional?
And callback arguments is not marked as undefined intentionally?